### DISCUSSION TITLE
c++ example about how to use tensorrt to speed up mxnet model for inference?

### DISCUSSION THREAD

**Discussion by IamMegatron2025** [apache/incubator-mxnet#14694] [Q&A]
Hello, I know [docs](https://mxnet.incubator.apache.org/versions/master/tutorials/tensorrt/inference_with_trt.html) about how to use tensorrt to speed up mxnet model using python for inference, but I can not find some docs for C++.
Similar problem is mentioned [here](https://github.com/apache/incubator-mxnet/issues/14341), but not solved.
Anyone can give some advises?

**Comment by frankfliu** [comment]
@mxnet-label-bot add [question, C++]

**Comment by frankfliu** [comment]
@IvyGongoogle Thanks for raising this question. For questions, please submit them on MXNet discussion forum (https://discuss.mxnet.io), where it will get a wider audience and allow others to learn as well.

@leleamol Do you have any example/docs for C++?

**Comment by IamMegatron2025** [comment]
need help...

**Comment by xizi** [comment]
need help too

**Comment by ou525** [comment]
need help too

**Comment by alicrop** [comment]
help
